vehicle/raw_vehicle_cmd_converter/src/csv_loader.cpp

```diff
@@ -100,10 +142,10 @@ std::vector<double> CSVLoader::getColumnIndex(const Table & table)
 double CSVLoader::clampValue(
   const double val, const std::vector<double> & ranges, const std::string & name)
 {
-  const double max_value = ranges.back();
-  const double min_value = ranges.front();
+  const double max_value = *std::max_element(ranges.begin(), ranges.end());
```

Comment:
Note: max = vec.back() min = vec.front() is common for current map format but I want this change for safe
```
         min ----> max
   min
   |
   max
```